sormas-ui/src/main/java/de/symeda/sormas/ui/contact/MergeContactsGrid.java

```diff
@@ -78,7 +79,11 @@ protected void buildColumns() {
 
 	@Override
 	protected List<MergeContactIndexDto[]> getItemForDuplicateMerging() {
-		return FacadeProvider.getContactFacade().getContactsForDuplicateMerging(criteria, ignoreRegion);
+		return FacadeProvider.getContactFacade()
+			.getContactsForDuplicateMerging(
+				criteria,
+				queryDetails.getResultLimit() != null ? queryDetails.getResultLimit() : CaseFacade.DUPLICATE_MERGING_LIMIT_DEFAULT,
```

Comment:
* Move the DUPLICATE_MERGING_LIMIT_DEFAULT constant to the AbstractMergeGrid class
* We have to check for a maximum here. Otherwise users can use the URL-based filter to enter much higher numbers and maybe stale the backend. Let's move the DUPLICATE_MERGING_LIMIT_MAX constant to AbstractMergeGrid as-well
* Same in MergeCasesGrid.getItemForDuplicateMerging
* Rename to getItemsForDuplicateMerging